src/rules/prefer-in-document.js

```diff
@@ -43,6 +45,24 @@ function usesToHaveLengthZero(matcherNode, matcherArguments) {
   return matcherNode.name === "toHaveLength" && matcherArguments[0].value === 0;
 }
 
+/**
+ * Extract the DTL query identifier from a call expression
+ *
+ * <query>() -> <query>
+ * screen.<query>() -> <query>
+ */
+function getDTLQueryIdentifierNode(callExpressionNode) {
+  if (!callExpressionNode || callExpressionNode.type !== "CallExpression") {
```

Comment:
I'm pretty sure we can use optional chaining here:

```suggestion
  if (callExpressionNode?.type !== "CallExpression") {
```

Reply:
I applied the suggestion but it turned out that ESLint does not recognize optional chaining syntax in this project.